airflow/utils/log/logging_mixin.py

```diff
@@ -135,7 +135,7 @@ def __init__(self, stream):
             self._use_stderr = False
 
         # StreamHandler tries to set self.stream
-        Handler.__init__(self)  # pylint: disable=non-parent-init-called
+        super(Handler).__init__()  # pylint: disable=bad-super-call
```

Comment:
And what do you think this change? @mik-laj

Reply:
Seem that will also call parent init, maybe we should keep this line in original.